algorithms.py

```diff
@@ -145,7 +178,100 @@ def configure(self, data, args):
         params = super(XgbGPUHistAlgorithm, self).configure(data, args)
         params.update({"tree_method": "gpu_hist", "gpu_id": 0})
         return params
+        
+class SkRandomForestAlgorithm(Algorithm):
+    def configure(self, data, args):
+        params = shared_params.copy()
+        if data.learning_task == LearningTask.REGRESSION:
+            params["objective"] = "regression"
+        elif data.learning_task == LearningTask.CLASSIFICATION:
+            params["objective"] = "binary"
+            params["scale_pos_weight"] = len(data.y_train) / np.count_nonzero(data.y_train)
+        elif data.learning_task == LearningTask.MULTICLASS_CLASSIFICATION:
```

Comment:
I don't think scikit-learn's random forest estimator makes distinction between binary classification and multi-class classification. So we should treat all classification tasks in a single `if` case.